tox.ini

```diff
@@ -1,6 +1,5 @@
 [tox]
 envlist =
-    py{36,37,38}-test
```

Comment:
The diff of this file seems wonky? you have deleted all the actual test lines no?